Source/WebKit/NetworkProcess/NetworkConnectionToWebProcess.cpp

```diff
@@ -1482,9 +1486,17 @@ void NetworkConnectionToWebProcess::logOnBehalfOfWebContent(IPC::DataReference&&
         osLog = adoptOSObject(os_log_create(subsystem, category));
     }
 
+    auto osLogPointer = osLog.get() ? osLog.get() : OS_LOG_DEFAULT;
+    auto logData = reinterpret_cast<const char*>(logString.data());
+
+#if HAVE(OS_SIGNPOST)
+    if (WTFSignpostHandleIndirectLog(osLogPointer, pid, logData))
+        return;
+#endif
```

Comment:
Would it make sense to only enable this code in internal builds, in order to avoid the sscanf for non internal builds?

Reply:
I think we need to be able to get signposts on internal builds that are emulating customer builds, so we don't want to do an internal build check here.

`WTFSignpostHandleIndirectLog` doesn't do anything unless the log subsystem is `com.apple.WebKit` and the category is `Signposts` so I expect that it will do nothing nearly all of the time.